src/components/projects/new/draft-card.svelte

```diff
@@ -0,0 +1,31 @@
+<script>
+  import { slide } from 'svelte/transition';
+  import Button from 'ui/button.svelte';
+  import fecha from 'fecha';
+  import {deleteDraft, chooseProject} from '@/store/new-project';
+
+  export let title = '';
+  export let date = '';
+
+  const choose = e => {
+    e.preventDefault();
+    chooseProject(date);
+  };
+</script>
+
+<div class="card" in:slide="{{duration: 300}}">
+  <div class="title">{title || 'Untitled'}</div>
+  {#if new Date(date) instanceof Date && !isNaN(new Date(date))}
```

Comment:
How can an object instantiated using `new Date()` _not_ be an `instanceof Date`?

Reply:
Simply: https://stackoverflow.com/a/1353711/7442294